### DISCUSSION TITLE
无法升级专业版

### DISCUSSION THREAD

**Discussion by CopyStar233** [getlantern/lantern-forum-cn#903] [General]
开发者你好，我已经支付了升级专业版的相关费用，但是没有自动升级成功，也没有在邮箱收到激活码，我的账号邮箱是pingfandeniu@foxmail.com，麻烦处理下，谢谢。
Hello developer, I have paid the relevant fees for upgrading to the professional version, but the automatic upgrade was not successful and I did not receive the activation code in my email. My account email is pingfandeniu@foxmail.com Could you please handle it? Thank you.

**Comment by CopyStar233** [comment]
![Screenshot_2024-09-27-14-02-32-22_3915bacb930634b7e206116f9dc9486f](https://github.com/user-attachments/assets/3e88d3d5-5f94-40fe-8fba-9eaafa7ef339)
这是账单

**Comment by CopyStar233** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@CopyStar233 您好. 好的, 我已经帮您提交问题给支付部门了. 他们会帮您处理. 谢谢您支持蓝灯!
